### ISSUE TITLE
Replace test assertion to use AssertJ

### ISSUE THREAD

**Issue by manusa** [eclipse/jkube#733]
### Description

The following test assertion is reported as a code smell by [SonarCloud](https://sonarcloud.io/project/issues?id=jkubeio_jkube&open=AXCb0U1sych7OcxsbwhB&resolved=false&types=CODE_SMELL):

https://github.com/eclipse/jkube/blob/682cc6f25ccf3e95e20832d1396fa3890f3a4e26/jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/SystemPropertyRegistryAuthHandlerTest.java#L63

Since we are transitioning our tests to use AssertJ, replace this assertion with the AssertJ equivalent:

```java
import static org.assertj.core.api.Assertions.assertThat;
/* ... */
assertThat(handler.create(RegistryAuthConfig.Kind.PUSH, null, "blubberbla:1611", s -> s))
  .isNull();
```

### Bonus

Replace the rest of assertions in this class (SystemPropertyRegistryAuthHandlerTest) with AssertJ.

**Comment by rbrishabh** [comment]
hey! I can help with this one too :)

**Comment by manusa** [comment]
[first-timers-only](https://github.com/eclipse/jkube/labels/first-timers-only) issues are intended for new contributors. If you have already committed to the project or are already working on an issue you shouldn't apply for these ones.

**Comment by danmikes** [comment]
Hello,
I think this would be a nice issue for me to pick up. However, it is my first time ever to contribute to open source. I have forked the repository and changed all asserts to use assertJ. Except in one test, where I ignored how to change it:
jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/assembly/AssemblyManagerCreateDockerTarArchiveTest.java
Should I try leave that one and make a pull request for the others? Or try to do it with some help from someone?
Should I push to the master branch of make a separate branch?
Please forgive my ignorance.
Daniel

**Comment by manusa** [comment]
Closed by #737
